postcss.config.js

```diff
@@ -1,23 +1,13 @@
 module.exports = {
   plugins: [
-    require('@tailwindcss/postcss7-compat'),
-    require('autoprefixer'),
-    require('postcss-import'),
-    require('postcss-flexbugs-fixes'),
-    require('postcss-preset-env')({
+    require("autoprefixer"),
+    require("postcss-import"),
+    require("postcss-flexbugs-fixes"),
+    require("postcss-preset-env")({
       autoprefixer: {
-        flexbox: 'no-2009'
+        flexbox: "no-2009",
       },
-      stage: 3
+      stage: 3,
     }),
-    // don't purge css in development
-    process.env.NODE_ENV === 'development' ? null :
-    require("@fullhuman/postcss-purgecss")({
-      content: [
-        "./app/**/*.haml",
-        "./app/javascript/**/*.js"
-      ],
-      defaultExtractor: content => content.match(/[^<>"{\.'`\s]*[^<>"{\.'`\s:]/g) || []
```

Comment:
I think this extractor was written to deal with the syntax of haml files (I think this was the source: https://medium.com/expected-behavior/purgecss-extractors-for-haml-a9dfe23a3504)
Can you verify that the correct classes are still extracted?

Reply:
In the most recent commit I added the same default extractor to the tailwind config, it should all work correctly. webpack-dev-server quickly reloads on change, which is reflected in the css.

Reply:
thanks :)